### PR TITLE
Add info regarding native datetime inputs.

### PR DESCRIPTION
**Pull request by ADmad**
Refs cakephp/cakephp#12934

### CODE REVIEW

File: en/appendices/4-0-migration-guide.rst [hunk 1]
```diff
@@ -181,5 +181,5 @@ View
 * ``FormHelper`` now generates HTML5 validation messages for fields marked as
   required in an entity's ORM table class. This feature can be toggled with the
   ``autoSetCustomValidity`` class configuration option.
-
-
+* ``FormHelper`` now generates native HTML5 input tags for datetime fields.
+  Check the :ref:`Form Helper <create-datetime-controls>`:: page for more details.
```
Comment (markstory) [inline en/appendices/4-0-migration-guide.rst]:
```suggestion
  Check the :ref:`Form Helper <create-datetime-controls>` page for more details.
```

File: en/views/helpers/form.rst [hunk 2]
```diff
@@ -1518,84 +1522,11 @@ way PHP handles data passed via file fields
...
-  the seconds select picker. Defaults to 24.
-
-* ``format`` - Applies to ``hour()``. The time format to use; either ``12`` or
-  ``24``. In case it's set to ``12`` the ``meridian`` select picker won't be
-  automatically displayed. It's up to you to either add it or provide means
-  to infer from the form context the right period of the day. Defaults to 24.
-
-* ``second`` - Applies to ``dateTime()`` and ``time()``. Set to ``true`` to
-  enable the seconds drop down. Defaults to ``false``.
-
```
Comment (markstory) [inline en/views/helpers/form.rst]: Do you have 2 headings in a with this block removed?

File: en/views/helpers/form.rst [hunk 3]
```diff
@@ -1627,68 +1547,19 @@ Output:
...
-        <option value="00">00</option>
-        ...
-        <option value="59">59</option>
-    </select>
+    <input type="datetime-local" name="registered" />
 
-To create datetime controls with custom classes/attributes on a specific select
-box, you can provide them as arrays of options for each component, within the
-``$options`` argument.
+The value for the input can be valid datetime string or ``DateTime`` instance.
```
Comment (markstory) [inline en/views/helpers/form.rst]:
```suggestion
The value for the input can be any valid datetime string or ``DateTime`` instance.
```

File: en/views/helpers/form.rst [hunk 4]
```diff
@@ -1697,52 +1568,19 @@ Creating Date Controls
...
-        ...
-        <option value="2018">2018</option>
-    </select>
-    <select name="registered[month]">
-        <option value="" selected="selected">Choose month...</option>
-        <option value="01">1</option>
-        ...
-        <option value="12">12</option>
-    </select>
+    <input type="datetime-local" name="registered" />
```
Comment (markstory) [inline en/views/helpers/form.rst]: In the text above you say that an input with the type=date will be created, but that is not what this example is showing.

File: en/views/helpers/form.rst [hunk 5]
```diff
@@ -1697,52 +1568,19 @@ Creating Date Controls
...
-                'month' => 'Choose month...'
-            ],
-            'day' => false,
-            'year' => [
-                'class' => 'cool-years',
-                'title' => 'Registration Year'
-            ]
-        ]);
-    ?>
+    <?= $this->form->dateTime('registered') ?>
```
Comment (markstory) [inline en/views/helpers/form.rst]: Should this be `date('registered')`?

File: en/views/helpers/form.rst [hunk 6]
```diff
@@ -1860,142 +1624,55 @@ Will output:
...
-
-    <select name="created[day]">
-        <option value="" selected="selected">Day</option>
-        <option value="01">1</option>
-        <option value="02">2</option>
-        <option value="03">3</option>
-        ...
-        <option value="31">31</option>
-    </select>
+    <input type="time" name="mob" />
```
Comment (markstory) [inline en/views/helpers/form.rst]: The type here doesn't match what the text above says will happen.

File: en/views/helpers/form.rst (hunk 2 above)
Comment (ADmad) [reply en/views/helpers/form.rst]: Not sure what you mean.

Comment (markstory) [reply en/views/helpers/form.rst]: From the diff it looks like there will be two adjacent headings. I can check that post merge though.

**Pull request merged**